tests/helpers.py

```
@@ -61,7 +61,9 @@ def run_in_executor(
         fut = self._loop.create_future()
         try:
             result = func(*args)
-        except Exception as exc:
+        except BaseException as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `BaseException`

(BLE001)